atoma-state/src/handlers.rs

```diff
@@ -766,6 +767,70 @@ pub(crate) async fn handle_state_manager_event(
     Ok(())
 }
 
+/// Handles an update to the number of compute units in a stack.
+///
+/// This function processes an update to the number of compute units in a stack by parsing the event data
+/// and updating the corresponding stack in the database.
+///
+/// # Arguments
+///
+/// * `state_manager` - A reference to the `AtomaStateManager` for database operations.
+/// * `stack_small_id` - The unique identifier of the stack to be updated.
+/// * `estimated_total_compute_units` - The estimated total number of compute units in the stack.
+/// * `total_compute_units` - The total number of compute units in the stack.
+///
+/// # Returns
+///
+/// * `Result<()>` - Ok(()) if the update was successful, or an error if something went wrong.
+///
+/// # Errors
+///
+/// This function will return an error if:
+/// * The database operation to update the stack number of compute units fails.
+/// * The Sui client operation to submit the claim funds for stacks transaction fails.
+#[instrument(
+    level = "info",
+    skip_all,
+    fields(stack_small_id, estimated_total_compute_units, total_compute_units)
+)]
+pub(crate) async fn handle_update_stack_num_compute_units(
+    state_manager: &AtomaStateManager,
+    stack_small_id: i64,
+    estimated_total_compute_units: i64,
+    total_compute_units: i64,
+) -> Result<()> {
+    info!(
+        target = "atoma-state-handlers",
+        event = "handle-claim-funds-for-stacks-tx-event",
```

Comment:
If this the correct event name? The function is for updating compute units.

Reply:
Nice catch